autoload/jedi.vim

```diff
@@ -437,7 +442,7 @@ function! jedi#show_call_signatures()
 endfunction
 
 
-function! jedi#clear_call_signatures()
+function! jedi#clear_call_signatures() abort
```

Comment:
Wow. This seems useful. Not exactly sure what the errors are where it would abort, but still.